src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="Surname">
+
+          Surname:
+          <input
```

Comment:
there shouldn't be any blank lines between parent and child tags, fix it everywere
```suggestion
        <label class="form-field" for="Surname">
          Surname:
          <input
```